docs/Deploy/Upgrading-Fleet.md

```diff
@@ -46,6 +46,8 @@ First, take the existing servers offline.
 Run database migrations:
 
 ```sh
+# make sure database servers currently offline or this will fail and might corrupt database
+# if ran this before taking offline, recommended rolling to backup
 fleet prepare db
 ```
 
```

Comment:
try using a giant purple block underneath the code example
and I'd suggest cutting all the words above the example except for like one sentence. Can talk about philosophy later (only as absolutely needed, cut as much as possible)

> Make sure database servers currently offline or this will fail and might corrupt database
> if ran this before taking offline, recommended rolling to backup